### PR TITLE
[Fix] Subscribe Section

### PR DESCRIPTION
**Pull request by santa116-1**

### CODE REVIEW

**Comment by Mr-HoDL58** [comment]
check broken links

**Review by b00f — APPROVED** [review]

File: website/index.html [hunk 1]
```diff
@@ -111,7 +108,11 @@ <h3>{% t dict.home.partners %}</h3>
     <div class="row p-4">
       <div class="col-md-4 text-center">
         <a href="https://sensifai.com" target="_blank">
+<<<<<<< HEAD
```
Comment (Ja7ad) [inline website/index.html:111]: Please remove this

File: website/index.html [hunk 2]
```diff
@@ -111,7 +108,11 @@ <h3>{% t dict.home.partners %}</h3>
...
+          <img alt="Sensifai" src="{{ site.url }}/assets/images/partners/sensifai.png" class="sensifai" />
+=======
```
Comment (Ja7ad) [inline website/index.html:113]: Please remove this

File: website/index.html [hunk 3]
```diff
@@ -111,7 +108,11 @@ <h3>{% t dict.home.partners %}</h3>
...
           <img alt="Sensifai" src="{{ site.url }}/assets/images/partners/sensifai.png" class="sensifai"/>
+>>>>>>> 929b44d198f1e4bed199a1ad39c4e9aa14a684c7
```
Comment (Ja7ad) [inline website/index.html:115]: Please remove this

**Review by Ja7ad — CHANGES_REQUESTED** [review]

**Pull request closed without merging**